src/backend/joseki.be/webapp/Controllers/AuditsController.cs

```diff
@@ -15,32 +16,54 @@ namespace webapp.Controllers
     [Route("api/audits")]
     public class AuditsController : Controller
```

Comment:
A general proposal.
Dates are often represented in different ways on different platforms (misc to-string serialization rules with timezone and without, unix-epoch, etc). 
What about exposing _dates_ from REST api as strings with a particular format (for example, `YYYYMMDD`) and then parsing it to `DateTime` object inside this controller methods?

Reply:
definitely, we should decide on a format and pass around strings. but question; what would happen if two scans run on same day? i remember we discussed this but don't recall we had a resolution.  I still believe we should keep the time portion.

Reply:
We agreed to persist the latest scan per day only